madgraph/iolibs/export_v4.py

```diff
@@ -6892,8 +6893,10 @@ def copy_standard_file(self):
         file = open(os.path.join(MG5DIR,\
                               'models/template_files/fortran/rw_para.f')).read()
 
-        includes=["include \'coupl.inc\'","include \'input.inc\'",
-                                              "include \'model_functions.inc\'"]
+        includes=["include \'../vector.inc\'",
+                  "include \'coupl.inc\'",
```

Comment:
This one is less clear, but I guess that it should also be removed correct?

Reply:
same as above, vector.inc is needed by coupl.inc